test/JDBC/input/BABEL-1625-vu-prepare.sql

```diff
@@ -0,0 +1,96 @@
+-- [BABEL-1625] expected value = 1912-11-25 12:24:32.0000000 +10:00
```

Comment:
Is there an existing file for datetimeoffset tests? If yes then can we add these tests directly in that file?

Reply:
we do have a separate datetimeoffset testcase file, but tests related to different date functions like dateadd(), datediff() etc. are added outside that file. Ex. of such files are- BABEL-3474, dateadd_internal_df etc. to name a few.